README.md

```diff
@@ -360,6 +362,30 @@ const kernel = gpu.createKernel(function(a, b) {
 }).setOutput([20]);
 ```
 
+### Adding strongly typed functions
+
+To strongly type a function you may use options.  Options take an optional hash values:
+`returnType`: optional, defaults to float, the value you'd like to return from the function
+`paramTypes`: optional, defaults to float for each param, a hash of param names with values of the return types
+
+Types: that may be used for `returnType` or for each property of `paramTypes`: 
+'Array'
```

Comment:
What is the meaning of "Array" type without number o elements? Can we give some example of that type?

Reply:
I also wann't to say that I don't deeply understand code so I can check on the the surface JS level only

Reply:
>What is the meaning of "Array" type without number o elements? Can we give some example of that type?

It is a standard array, passed from javascript to webgl.